bot/slash_premint.go

```diff
@@ -46,14 +47,13 @@ func premintSlashCommand(ctx context.Context, logger *zap.SugaredLogger, databas
 			// TODO: Validate ETH address
 			withAddress = true
 			logger.Info("Checking premint status with the ETH wallet address")
-			addressOption := strings.ToLower(i.ApplicationCommandData().Options[0].StringValue())
-			address := ""
+			addressOption := i.ApplicationCommandData().Options[0].StringValue()
 			// Check if the address is actually an ENS name
 			if !strings.HasPrefix(addressOption, "0x") {
-				address = strings.ToLower(infuraClient.GetAddressFromENSName(addressOption))
+				address = infuraClient.GetAddressFromENSName(addressOption)
```

Comment:
Didn't actually test this... I wonder if infura needed the address in all lowercase?